openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/weightings/AdditionWeightingTest.java

```diff
@@ -0,0 +1,33 @@
+package org.heigit.ors.routing.graphhopper.extensions.weightings;
+
+import com.graphhopper.routing.util.EncodingManager;
+import org.heigit.ors.routing.graphhopper.extensions.ORSDefaultFlagEncoderFactory;
+import org.heigit.ors.routing.graphhopper.extensions.flagencoders.CarFlagEncoder;
+import org.heigit.ors.routing.graphhopper.extensions.flagencoders.FlagEncoderNames;
+import org.heigit.ors.routing.graphhopper.extensions.weighting.AdditionWeighting;
+import org.heigit.ors.routing.graphhopper.extensions.weighting.ConstantWeighting;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class AdditionWeightingTest {
+    private final EncodingManager encodingManager;
+    private final CarFlagEncoder flagEncoder;
+
+    public AdditionWeightingTest() {
+        encodingManager = EncodingManager.create(new ORSDefaultFlagEncoderFactory(), FlagEncoderNames.CAR_ORS, 4);
+        flagEncoder = (CarFlagEncoder)encodingManager.getEncoder(FlagEncoderNames.CAR_ORS);
+    }
+
+    @Test
+    public void sumOfConstants () {
+        ConstantWeighting const1 = new ConstantWeighting(1,10);
+        ConstantWeighting const2 = new ConstantWeighting(2,20);
+        ConstantWeighting const3 = new ConstantWeighting(3,30);
+        ConstantWeighting factor = new ConstantWeighting(10, 100);
+
+        ConstantWeighting[] weightingsShort = {const1, const2, const3};
+        AdditionWeighting additionWeightingShort = new AdditionWeighting(weightingsShort,factor,flagEncoder);
+        assertEquals(60, additionWeightingShort.calcWeight(null, false, 0),0.0001);
```

Comment:
How about testing for `calcMillis` as well?

Reply:
Yes, now that we understand what `calcMillis` should do, I can add a test.

Reply:
I just noticed the useless suffix "Short". Apparently the IDE did some renaming without me noticing it.

Reply:
Yeah, I was a little bit surprised by this name too :)